training/tests/super_class.py

```diff
@@ -0,0 +1,44 @@
+import unittest
+
+from training.app import app
+from training.extensions import db
+
+
+class setUpAndTearDown(unittest.TestCase):
```

Comment:
Class names should start with capital letter

Reply:
Changed!